### DISCUSSION TITLE
Discussion: Reordering ListView items shouldn't require AddDeleteThemeTransition

### DISCUSSION THREAD

**Discussion by FrayxRulez** [microsoft/microsoft-ui-xaml#7487] [General]
# Discussion: Reordering ListView items shouldn't require AddDeleteThemeTransition

I don't see any real connection between `AddDeleteThemeTransition` and reordering items animation, but I would expect `ReorderThemeTransition` to handle this scenario.
I have a chat list, where items are moved pretty quickly, and `AddDeleteThemeTransition` just doesn't fit, because it is too attractive for the eye to play continuously. Still, users should be able to reorder the first chats in the list (that can be locked/pinned).
The problem is: if I remove `AddDeleteThemeTransition`, reordering looks pretty much broken:
![animation](https://media.giphy.com/media/LhYByyiLBmLVLgUVX2/giphy.gif)

So yeah, it would be great if this animation could be handled by a different Transition.

**Comment by StephenLPeters** [comment]
@stmoy fyi

**Comment by stmoy** [comment]
@FrayxRulez - thanks for filing this issue! Agreed that the GIF looks pretty broken.

ListView has four transitions by default:

``` xml
<TransitionCollection>
  <AddDeleteThemeTransition />
  <ContentThemeTransition />
  <ReorderThemeTransition />
  <EntranceThemeTransition IsStaggeringEnabled="False" />
</TransitionCollection>
```

Can you confirm which transitions your list has? I agree that `AddDeleteThemeTransition` shouldn't break the reordering, but the ThemeTransition types are mysterious and interconnected in ways that aren't obvious.

**Comment by FrayxRulez** [comment]
My collection has all of the default ones, but `AddDeleteThemeTransition`!

**Comment by stmoy** [comment]
@FrayxRulez - thank you for confirming. That is very strange indeed. I don't have any insightful workarounds offhand - this just seems like a bug to me. 🤷‍♂️
